src/bootstrap.tsx

```diff
@@ -687,6 +688,11 @@ const renderApp = async () => {
       i18n.changeLanguage(defaultLanguage);
     }
 
+    const printAppTemplates
+      = appConfig?.clientConfig?.printApp;
+
+    store.dispatch(setPrintApp(printAppTemplates));
```

Comment:
Please wrap the dispatch in a condition. So it will be applies to the store if set in the configuration only.